index/scorch/scorch.go

```diff
@@ -507,8 +532,11 @@ func (s *Scorch) StatsMap() map[string]interface{} {
 	m["num_items_introduced"] = m["TotIntroducedItems"]
 	m["num_items_persisted"] = m["TotPersistedItems"]
 	m["num_recs_to_persist"] = m["TotItemsToPersist"]
-	m["num_bytes_used_disk"] = m["CurOnDiskBytes"]
-	m["num_files_on_disk"] = m["CurOnDiskFiles"]
+	// total disk bytes found in index directory inclusive of older snapshots
+	m["num_bytes_used_disk"] = numBytesUsedDisk
+	// total disk bytes by index exclusive of older snapshots
+	m["num_bytes_used_disk_index"] = numBytesOnDiskByIndex
```

Comment:
This stat seems more like "num_bytes_used_disk" to me.
The older num_bytes_used_disk can be "num_bytes_on_disk" - what do you think?

Reply:
older stats name could be something like a "total_num_bytes_on_disk" etc, but we can't change stats due to backward compatibility as we don't know whether anyone actually started using it or not ?

Reply:
Yes, we certainly cannot rename stat keys as they are just strings passed through to users.